index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather App</title>
+    <link rel="stylesheet" href="weather.css"/>
+    <script src="index.js" defer></script>
+</head>
+<body>
+    <header><h1>Weather App</h1>
+   
+    <form>
+       <label>
+        Pick a Location
+        <input type="text" name="location" >
+        <input type="submit" name="Get Weather" value="Get Weather">
+       </label>
+    </form>
+  
+    </header>
+
+ <main>
+    <p class="choose">Choose a location to view the weather.</p>
+ </main>
+
+ <aside class="convert">
```

Comment:
The converter needs the appropriate IDs mentioned in the tests. Check the test results to see which IDs need to go on which elements